modules/1_opencv/src/opencv_coins.cpp

```diff
@@ -3,12 +3,32 @@
 using namespace cv;
 
 unsigned countCoins(const Mat& img) {
-    Mat gray, thresh;
-    cvtColor(img, gray, COLOR_BGR2GRAY);
-    threshold(gray, thresh, 0, 255, THRESH_BINARY_INV + THRESH_OTSU);
-    // Remove unfilled black holes by morphological closing
-    morphologyEx(thresh, thresh, MORPH_CLOSE, Mat::ones(3, 3, CV_8U), Point(-1, -1), 3);
-
-    // TODO: implement an algorithm from https://docs.opencv.org/master/d3/db4/tutorial_py_watershed.html
-    CV_Error(Error::StsNotImplemented, "countCoins");
+	Mat gray, thresh;
+	cvtColor(img, gray, COLOR_BGR2GRAY);
+	threshold(gray, thresh, 0, 255, THRESH_BINARY_INV + THRESH_OTSU);
+	morphologyEx(thresh, thresh, MORPH_CLOSE, Mat::ones(3, 3, CV_8U), Point(-1, -1), 3);
+	dilate(thresh, thresh, 3);
+	distanceTransform(thresh, thresh, DIST_L2, 5);
+	
+
+	double maxValue, minValue;
+	Point maxLoc, minLoc;
+	minMaxLoc(thresh, &minValue, &maxValue, &minLoc, &maxLoc);
+	threshold(thresh, thresh, 0.7*maxValue, 255, 0);
+	
+
+	thresh.convertTo(thresh, CV_8U);
+	std::vector<std::vector<Point> > contours;
+	findContours(thresh, contours, RETR_EXTERNAL, CHAIN_APPROX_SIMPLE);
+	
+	int Sum = 0;
+	for (int i = 0; i < contours.size(); i++){
+		double radius = arcLength(contours[i], true) /( 2 * CV_PI);
+		if (radius > 15 ){
+		  Sum += 2;
+        }
+		else Sum += 1;
+	}
```

Comment:
Please fix indentation:
```cpp
for (int i = 0; i < contours.size(); i++) {
    double radius = arcLength(contours[i], true) /( 2 * CV_PI);
    if (radius > 15 ) {
        Sum += 2;
    }
    else {
        Sum += 1;
    }

```

Reply:
I didn't really understand what it was indentation

Reply:
Number of whitespaces before `double radius = arcLength(contours[i], true) /( 2 * CV_PI);`

Reply:
See https://github.com/dkurt/openvino_practice/pull/95/files